torch/testing/_internal/common_fsdp.py

```diff
@@ -687,6 +687,6 @@ def _collect_total_grad_norm_local(model, norm_type):
     else:
         total_norm = 0.0
         for p in model.parameters():
-            local_norm = torch.linalg.norm(p.grad, norm_type, dtype=torch.float32)
+            local_norm = torch.linalg.vector_norm(p.grad, norm_type, dtype=torch.float32)
```

Comment:
The [CI error](https://github.com/pytorch/pytorch/runs/6376569510?check_suite_focus=true
) was a bug in their testing code (!!)
They were calling `linalg.norm` with a 2D tensor with `norm_type` (i.e. `ord`) equal to 2.5. Now, as it says in `linalg.norm` docs:
>If `dim=None` and `ord !=None`, `A` must be 1D or 2D.

In this case, when they were passing `ord=2`, it was computing the incorrect norm (it was computing the 
**matrix** 2-norm). Now, when they passed `ord=2.5`, the correct testing of this op identified that this was incorrect.